docs/_data/docs.yml

```diff
@@ -46,5 +46,6 @@
 - title: Meta
   docs:
   - contributing
+  - development
```

Comment:
I think I'd like this to be called `maintaining` or something like that. @jekyll/affinity-team-captains, how do you feel about putting your docs on the jekyllrb.com website?